src/db/seeds/crafting.ts

```
@@ -0,0 +1,151 @@
+import { db } from "../index";
+import { recipes, recipeIngredients } from "../schema";
+import { eq } from "drizzle-orm";
```

ESLint

[error] 'eq' is defined but never used.

(@typescript-eslint/no-unused-vars)